XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -24,7 +24,7 @@
       "Description": "",
       "Items": [
         {
-          "UniqueId": "UICommand",
+          "UniqueId": "XamlUICommand",
```

Comment:
Please remove the reference to "StandardUICommand - API" if it isn't pertinent to this sample.